Modules/LearningSequence/classes/class.ilLearningSequenceExporter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @author Daniel Weise <daniel.weise@concepts-and-training.de>
+ */
+class ilLearningSequenceExporter extends ilXmlExporter
+{
+	public function init()
+	{
+		global $DIC;
```

Comment:
Kontrollieren wir die Erstellung? dann könnte das auch in den Constructor...

Reply:
Leider nein